tests/integration/create_token.py

```diff
@@ -88,6 +88,7 @@ def _get_payload_with_params(
         payload_vars["exp"] = payload_vars["iat"] + float(3600)  # one hour from now
         payload_vars["jti"] = str(uuid4())
         payload_vars["case_id"] = str(uuid4())
+        payload_vars["response_expires_at"] = "2023-05-18T10:38:13+00:00"
```

Comment:
Will this work past 18th?. I suggest just setting where ever we need these to like 
```py
(datetime.now(tz=timezone.utc) + timedelta(days=1)).isoformat()
```

Reply:
I was wondering about that too, dynamic date is much better, thanks.